NuGet.config

```diff
@@ -3,8 +3,9 @@
   <packageSources>
     <clear />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
-    <add key="darc-pub-dotnet-runtime-cf258a1-1" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-runtime-cf258a14-1/nuget/v3/index.json" />
```

Comment:
Seeing similar removals in dotnet/efcore#23129 and dotnet/extensions#3659. Not sure of the correct fix. @mmitche and @wtgodbe can you help❔

Reply:
Verifying. I think something wacky happened.